client/src/components/OngoingGoal.jsx

```diff
@@ -11,15 +11,15 @@ const OngoingGoal = ({ goal }) => {
   };
   return (
     <div>
-      <Wrapper>
+      <Wrapper onClick={onClickHandler}>
```

Comment:
Sometimes will pop up `userInfo.onGoingGoals is not iterable` error. Can ignore if you can't debug this.